### ISSUE TITLE
[FR] Add icon in front of headings to zoom in via click

### ISSUE THREAD

**Issue by nousernameavailableanymore** [vslinko/obsidian-zoom#112]
In lists you can zoom in by clicking on the little dot in front of a list item.

It would be awesome if there was a similar icon added in front of headings, which can be clicked to zoom in to headings (at the moment it seems the "Zoom in" command needs to be used).

**Comment by Enteleform** [comment]
+1

This would also fix the inability to zoom to headings when using the reading view.

**Comment by alexslade** [comment]
And the inability to zoom on iPad

Edit: It *is* possible to zoom on iPad, my mistake. But as above, this can only be done in edit mode.

**Comment by diego898** [comment]
Yeah currently you can zoom in on a bullet/sub-list by clicking the bullet icon. You can only zoom in on headings by keyboard shortcut. Would be great to be able to click the heading itself (or an icon to the left) to be able to zoom into it as well.
